common/shield_exceptions.cc

```diff
@@ -40,6 +40,18 @@ bool IsBlockedResource(const GURL& gurl) {
 
 bool IsWhitelistedFingerprintingException(const GURL& firstPartyOrigin,
     const GURL& subresourceUrl) {
+  // Always allow embeds from public.tableau.com while fingerprinting
+  // protections are being reworked to need less exceptions.
+  static const std::vector<URLPattern> embed_exceptions = {
+    URLPattern(URLPattern::SCHEME_ALL, "https://public.tableau.com/*"),
+    URLPattern(URLPattern::SCHEME_ALL, "https://www.arcgis.com/*"),
+  };
+  for (const auto exception : embed_exceptions) {
```

Comment:
I guess this is ok because it's temporary anyway, but ideally it would be grouped with the others and use `SetMatchAllURLs` for the primary pattern